src/v2/index.ts

```diff
@@ -53,24 +52,24 @@ if (IS_PROD) {
 // This date is used to check if the message's been created before the bot's started
 export const startTime = new Date();
 loadLastDecayFromDB();
-
 const client = new Client({
   intents: [
-    Intents.FLAGS.GUILDS,
-    Intents.FLAGS.GUILD_MEMBERS, // Privileged Intent
-    Intents.FLAGS.GUILD_BANS,
-    Intents.FLAGS.GUILD_EMOJIS_AND_STICKERS,
-    Intents.FLAGS.GUILD_INTEGRATIONS,
-    Intents.FLAGS.GUILD_WEBHOOKS,
-    Intents.FLAGS.GUILD_INVITES,
-    Intents.FLAGS.GUILD_VOICE_STATES,
-    Intents.FLAGS.GUILD_PRESENCES, // Privileged Intent
-    Intents.FLAGS.GUILD_MESSAGES,
-    Intents.FLAGS.GUILD_MESSAGE_REACTIONS,
-    Intents.FLAGS.GUILD_MESSAGE_TYPING,
-    Intents.FLAGS.DIRECT_MESSAGES,
-    Intents.FLAGS.DIRECT_MESSAGE_REACTIONS,
-    Intents.FLAGS.DIRECT_MESSAGE_TYPING,
+    'Guilds',
+    'GuildMembers',
+    'GuildBans',
+    'GuildEmojisAndStickers',
+    'MessageContent',
```

Comment:
Note: This is new and will probably require @ljosberinn to update bot permissions on his end

Reply:
happy to do that but uh where 👀 is this server settings side on discord or bots permissions on the page somewhere?

Reply:
It's on the discord developer portal, under app intents or something.

Reply:
this is already active 🎉 
![image](https://github.com/user-attachments/assets/4ba4db23-a179-460a-b58a-9d31c04ad198)